deploy.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/bash
+apt-get install sysv-rc-softioc
+if [ ! -d /opt/conda_envs/services/ ]; then conda create -p /opt/conda_envs/services python=3.5; fi
+source activate services
+conda install pymongo ujson tornado jsonschema yaml pytz doct
+pip install requests
+mkdir /services
+cd /services
+mkdir -p /epics/iocs/
+
+t clone https://github.com/NSLS-II/amostra.git
```

Comment:
We could pull amostra from conda with all its dependencies instead of doing this